documenting synthetic test id (#95)
instana/openapi: spec/descriptions/createSyntheticTest.md

**Note:** The **DNSAction** Synthetic type is not supported.

## Optional Parameters:

> - **id** Users are allowed to specify their own id for the test. A test id can contain letters, numbers, underscores, colons and hyphens. Max length is 128.

Reviewer: ```suggestion
- **id** Users may specify their own id for the test. A test id can contain letters, numbers, underscores, colons and hyphens. The maximum length of the test id is 128.
```

Reviewer: @andreachiavegatto should we show the allowable characters like Aa-Zz, 0-9, etc?  Also, are UTF-8 characters supported?  Some letters have accents or umlauts in other languages.  Does it affect the length?

Author: I copied what we have for the [credentials](https://github.com/instana/openapi/blob/master/spec/descriptions/createSyntheticCredential.md?plain=1#L5) documentation and just adjusted for the characters we support
